### PR TITLE
Reorganize

### PR DESCRIPTION
**Pull request by synixebrett**
Reorganizes the site to be much clearer, adds driving and convoy tactics

### CODE REVIEW

File: guides/roles/driver.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### Unarmored Transport
+Vehicles that are used for high speed, high capacity, or long-range transportation. They should never be expected to be used in combat and are rarely equipped with any defenses.
+
+*Examples* Prowler, HEMTT, Civilian Vehicles
+
+### Armored Transport
+Vehicles that are used for high mobility light combat transportation. Usually equipped with some mounted weapon for defense.
+
+*Examples* HMMWV, Hunter, Strider
```
Comment (nameless) [inline guides/roles/driver.md]: The last two are all [MRAPs](https://en.wikipedia.org/wiki/Oshkosh_M-ATV) though...

In the game the Fennek is meant to act as an MRAP aswell.

File: index.html [hunk 2]
```diff
@@ -10,13 +10,22 @@
...
+      <a href="#/guides/tactics/readme.md">Tactics</a>
+      <a href="#/guides/roles/readme.md">Roles</a>
+      <a href="#/guides/makers/readme.md">Maker Guides</a>
+    </nav>
   <div id="app"></div>
   <script>
     window.$docsify = {
       name: 'Synixe',
       repo: 'Synixe/Documents',
+      logo: 'https://cdn.discordapp.com/avatars/403101852771680258/3851b8701c42218e485dae9fe32d51e9.webp?size=512',
```
Comment (nameless) [inline index.html]: Using a URL for images is unacceptable no?

File: guides/roles/driver.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+### Mine Resistant Ambush Protected (MRAP)
+Large well-protected vehicles that offer the most safety during combat, at the cost of slower and louder operation that is easier to detect.
+
+*Examples* RG-31, Titan
```
Comment (nameless) [inline guides/roles/driver.md]: We dont even have the Titan in the mod anymore

File: guides/roles/driver.md (hunk 1 above)
Comment (nameless) [reply guides/roles/driver.md]: Are we using ACE vehicle names or ARMA vehicle names?

File: guides/tactics/convoy.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
+# Convoys
+
+## Required Reading
+
+This document assumes the reader is already familiar with [Driving Roles](guides/roles/driver.md).
+
+## Operating
+
+### Speed
+There are 3 main speeds in a convoy: *Full* - 80 KM/h  |  *Half* - 40 KM/h  |  *City* - 25 KM/h
```
Comment (nameless) [inline guides/tactics/convoy.md]:
```suggestion
There are 3 main speeds in a convoy: *Full* - 80 km/h  |  *Half* - 40 km/h  |  *City* - 25 km/h  
```
Typically KM/h is not capitalised and such capitalisation should only apply to MPH.

File: index.html (hunk 2 above)
Comment (synixebrett) [reply index.html]: This is a link to the actual server icon, which is not at risk of being deleted

Comment (synixebrett) [reply index.html]: This is a link to the actual server icon, which is not at risk of being deleted

File: guides/roles/driver.md (hunk 1 above)
Comment (synixebrett) [reply guides/roles/driver.md]: Going to use ACE names

File: index.html (hunk 2 above)
Comment (nameless) [reply index.html]: Still could have used a file path though

**Pull request merged**